### DISCUSSION TITLE
Loading state for buttons using `isLoading` prop

### DISCUSSION THREAD

**Discussion by burhanuday** [heroui-inc/heroui#596] [Feature requests]
The button component has a disabled state but not a loading state

**Describe the solution you'd like**
The loading state for buttons is very convenient to have. For example, when submitting a form, you can set prop `isLoading={true}` on the button which would render a spinner inside the button

<img width="108" alt="image" src="https://user-images.githubusercontent.com/25934701/151685971-4a068035-0fc3-4b0c-b37c-9f4641061973.png">
 
 Checkout [ChakraUI](https://chakra-ui.com/docs/form/button#button-loading-state) and [MaterialUI](https://mui.com/components/buttons/#loading-button)
 
 We already have a `Loading` component which can be rendered inside the Button if isLoading prop is true

**Additional context**
I'd like to pick this task up once it gets green ligh

**Comment by jm-janzen** [comment]
Seems to have a loading state now. See docs [here](https://nextui.org/docs/components/button#loading). 🙂
